src/sinks/pulsar.rs

```diff
@@ -77,6 +81,14 @@ pub struct PulsarSinkConfig {
     partition_key_field: Option<String>,
 }
 
+/// Event batching behavior.
+#[configurable_component]
+#[derive(Clone, Copy, Debug, Default)]
+pub struct BatchConfig {
+    /// The maximum size of a batch, in events, before it is flushed.
+    pub max_events: Option<u32>,
```

Comment:
Can we make this `batch_size`? Even though events is the right Vector-specific name for a message, I think that the Pulsar terminology will help people here when they're looking for this setting.

Reply:
Fixed.